.github/workflows/tests.yaml

```diff
@@ -74,7 +74,7 @@ jobs:
     - name: Setup Ruby
       uses: ruby/setup-ruby@v1
       with:
-        ruby-version: 2.6
+        ruby-version: ${{ env.ruby_version }}
         bundler-cache: true # runs 'bundle install' and caches installed gems automatically
```

Comment:
Dang, I'm just noticing the whole step below this one is unnecessary. Feel free to remove that while you're in here, if you want.

Reply:
The `Bundle Install` step is unnecessary? Doesn't that install cocoapods?

Reply:
Yeah. [setup-ruby does that automatically](https://github.com/ruby/setup-ruby#caching-bundle-install-automatically), and better, since it uses a cache.

Reply:
TIL! Cool will do.